src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-list/rgw-bucket-list.component.html

```diff
@@ -1,3 +1,6 @@
+<cd-alert-panel *ngIf="isTimeOut"
+                type="warning"
+                i18n>PLEASE RELOAD THE BUCKET LIST AS THERE IS SOME POSSIBLE STALE DATA</cd-alert-panel>
```

Comment:
Please don't use all caps and change the phrasing.
First inform the user the data might be stale and after inform that he can manually reload.

Reply:
'The bucket list data might me stale, can be manually reloaded' is this fine?